Fika.Core/Coop/Players/ObservedCoopPlayer.cs

```diff
@@ -868,6 +868,7 @@ public void InitObservedPlayer(bool isDedicatedHost)
 				playerTraverse.Field("_vaultAudioController").SetValue(null);
 				playerTraverse.Field("_sprintVaultAudioController").SetValue(null);
 				playerTraverse.Field("_climbAudioController").SetValue(null);
+				RaycastCameraTransform = playerTraverse.Field<Transform>("_playerLookRaycastTransform").Value;
```

Comment:
Is there any effects to this that I should be aware of?

Reply:
I think it should be fine, considering this is also initialized on a player.